great_expectations/expectations/metrics/column_aggregate_metrics/column_mean.py

```diff
@@ -31,7 +32,15 @@ def _sqlalchemy(cls, column, **kwargs):
     @column_aggregate_partial(engine=SparkDFExecutionEngine)
     def _spark(cls, column, _table, _column_name, **kwargs):
         """Spark Mean Implementation"""
-        types = dict(_table.dtypes)
-        if types[_column_name] not in ("int", "float", "double", "bigint"):
-            raise TypeError("Expected numeric column type for function mean()")
+        column_data_type = _table.schema[_column_name].dataType
+        if type(column_data_type) not in (
```

Comment:
@TrangPham I believe that we still need to import `types` from pyspark compatibility.  Thanks.

Reply:
It seems to be required:

<img width="1072" alt="Screenshot 2023-05-15 at 4 51 56 PM" src="https://github.com/great-expectations/great_expectations/assets/2117313/724ac379-2e83-499b-a3b4-2524cbdbbb40">

Reply:
Without type() here, it would return "Decimal(38,18)". The type of which is "Decimal". What are your suggestions here?